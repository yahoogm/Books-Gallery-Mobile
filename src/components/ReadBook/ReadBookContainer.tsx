import ReadBook from './ReadBook';
import {ReadBookProps} from './types';

const ReadBookContainer: React.FC<ReadBookProps> = ({
  refReadBook,
  setShowBook,
  showBook,
  isbn,
}) => {
  return (
    <ReadBook
      refReadBook={refReadBook}
      setShowBook={setShowBook}
      showBook={showBook}
      isbn={isbn}
    />
  );
};

export default ReadBookContainer;
