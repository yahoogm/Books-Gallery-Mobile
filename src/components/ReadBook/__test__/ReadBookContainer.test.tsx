import {createRef} from 'react';
import {render} from '@testing-library/react-native';
import ReadBookContainer from '../ReadBookContainer';

describe('Read Book Container', () => {
  it('match snapshot when renders', () => {
    const refReadBook = createRef<any>();
    const mockSetShowBook = jest.fn();

    const renderComponent = render(
      <ReadBookContainer
        refReadBook={refReadBook}
        setShowBook={mockSetShowBook}
        isbn={2020400192}
        showBook={false}
      />,
    );

    const tree = renderComponent.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
