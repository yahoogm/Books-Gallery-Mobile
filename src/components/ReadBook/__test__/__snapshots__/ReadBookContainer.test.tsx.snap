// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Read Book Container match snapshot when renders 1`] = `
<RCTScrollView
  data-style=""
  dataSet={
    {
      "style": "",
    }
  }
  style={
    [
      undefined,
      {},
      undefined,
    ]
  }
>
  <View />
</RCTScrollView>
`;
